src/scripts/main.js

```diff
@@ -1,3 +1,43 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+
+const promise1 = new Promise((resolve, reject) => {
+  logo.addEventListener('click', () => {
+    resolve();
+  });
+});
+
+const promise2 = new Promise((resolve, reject) => {
+  setTimeout(() => {
+    reject(Error);
+  }, 3000);
+});
+
+const error = () => {
+  const div = document.createElement('div');
+
+  div.classList.add('error-message');
+  div.innerText = `Promise was rejected!`;
+
+  document.body.appendChild(div);
+};
+
+const success = () => {
+  const div = document.createElement('div');
+
+  div.classList.add('message');
+  div.innerText = `Promise was resolved!`;
+
+  document.body.appendChild(div);
+};
```

Comment:
This function has a lot in common. They only differs with message and inner text

Reply:
Let's write an `addMessage` function that takes two args